Objects/clinic/unicodeobject.c.h

```diff
@@ -582,7 +582,7 @@ PyDoc_STRVAR(unicode_strip__doc__,
 "strip($self, chars=None, /)\n"
 "--\n"
 "\n"
-"Return a copy of the string with leading and trailing whitespace remove.\n"
+"Return a copy of the string with leading and trailing whitespace removed.\n"
```

Comment:
This file is automatically generated using argument clinic, you need to regenerate it using `make regen-all` or `make clinic`

Reply:
executed make clinic